src/JobQueue/Message/MessageSerializationInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Icecave\Overpass\JobQueue\Message;
+
+use Icecave\Overpass\JobQueue\Exception\InvalidMessageException;
+
+interface MessageSerializationInterface
+{
+    /**
+     * Serialize a task request message.
+     *
+     * @param Request $request
+     *
+     * @return string
+     */
+    public function serializeRequest(Request $request);
```

Comment:
Did we consider calling `Request` as `Job` instead when we talked about this stuff?